src/misc.ts

```diff
@@ -594,11 +594,18 @@ export const findById = <T extends Node>(id: string, list?: T[]) =>
 export const COLOR_WARNING = "#FBE5AC";
 export const COLOR_WARNING_DARK = "#3E2F0A";
 
+export type PillStatusType =
+  | "error"
+  | "warning"
+  | "info"
+  | "success"
+  | "generic";
+
```

Comment:
You can use:
`import { PillColor } from "@saleor/macaw-ui";`
although this pushes back macaw migration so it's optional